fairer-iOS/fairer-iOS/Screens/OnboardingName/OnboardingNameViewController.swift

```diff
@@ -0,0 +1,163 @@
+        textField.attributedPlaceholder = NSAttributedString(string: TextLiteral.onboardingNameViewControllerTextFieldPlaceholder, attributes: attributes)
+        textField.layer.cornerRadius = 8
+        textField.autocorrectionType = .no
+        textField.autocapitalizationType = .none
+        textField.leftView = UIView(frame: CGRect(x: 0, y: 0, width: 24, height: 58))
+        textField.leftViewMode = .always
+        textField.setClearButton()
+        return textField
+    }()
+    private let nameDoneButton: MainButton = {
+        let button = MainButton()
+        button.title = TextLiteral.onboardingNameViewControllerDoneButtonText
+        button.isDisabled = true
+        button.addTarget(self, action: #selector(didTapDoneButton), for: .touchUpInside)
+        return button
+    }()
+    private let disableLabel: UILabel = {
+        let label = UILabel()
+        label.text = TextLiteral.onboardingNameViewControllerDisableLabel
+        label.textColor = .negative20
+        label.font = .body2
+        label.layer.opacity = 0
+        return label
+    }()
+    
+    // MARK: - life cycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupDelegation()
+        setupNotificationCenter()
+    }
+    
+    override func configUI() {
+        super.configUI()
+    }
+    
+    override func render() {
+        view.addSubview(nameLabel)
+        nameLabel.snp.makeConstraints {
+            $0.leading.equalTo(24)
+            $0.top.equalToSuperview().offset(111)
+        }
+        
+        view.addSubview(nameTextField)
+        nameTextField.snp.makeConstraints {
+            $0.leading.equalTo(24)
+            $0.trailing.equalTo(-24)
+            $0.top.equalTo(nameLabel.snp.bottom).offset(16)
+            $0.height.equalTo(58)
+        }
+        
+        view.addSubview(nameDoneButton)
+        nameDoneButton.snp.makeConstraints {
+            $0.leading.equalTo(24)
+            $0.trailing.equalTo(-24)
+            $0.bottom.equalToSuperview().offset(-36)
+        }
+        
+        view.addSubview(disableLabel)
+        disableLabel.snp.makeConstraints {
+            $0.leading.equalTo(24)
+            $0.top.equalTo(nameTextField.snp.bottom).offset(8)
+        }
+    }
+    
+    // MARK: - functions
+    
+    private func setupDelegation() {
+        nameTextField.delegate = self
+    }
+    
+    @objc private func didTapDoneButton() {
+        if nameTextField.text!.hasCharacters() {
+            nameTextField.layer.borderWidth = 0
+            disableLabel.layer.opacity = 0
+            
+            // TODO: - userdefault에 이름 저장
+            
+        } else {
+            nameTextField.layer.borderWidth = 1
+            nameTextField.layer.borderColor = UIColor.negative20.cgColor
+            disableLabel.layer.opacity = 1
+            nameDoneButton.isDisabled = true
+        }
+    }
+    
+    @objc private func keyboardWillShow(notification:NSNotification) {
```

Comment:
p5;

```suggestion
    @objc private func keyboardWillShow(notification: NSNotification) {
```
요기 한 칸만 띄어줄 수 있나요... ㅎㅎ

Reply:
[7923939](https://github.com/fairer-iOS/fairer-iOS/pull/15/commits/79239398bf59d8df61c6b592d82fedb78082b477)
수정해서 반영했습니다!